productions/templates/productions/show.html

```diff
@@ -28,20 +24,20 @@
 {% block extra_head %}
     {{ carousel.media }}
     <meta name="twitter:card" content="summary_large_image" />
-    <meta name="twitter:site" content="@demozoo">
+    <meta name="twitter:site" content="@demozoo" />
```

Comment:
What's the deal with the self-closing slashes? I thought those died along with XHTML...?

Reply:
Oh that's me being strict. As a rule it's neat to always close HTML elements whether it's "necessary" or not. We also don't leave `<li>` or `<p>` elements dangling -even though that's is not considered mandatory by HTML5-type parsers.